core/src/epicli/data/common/ansible/playbooks/roles/helm_charts/files/system/node-exporter/values.yaml

```diff
@@ -0,0 +1,236 @@
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry and imagePullSecrets
+##
+global:
+  # imageRegistry: myRegistryName
+  # imagePullSecrets:
+  #   - myRegistryKeySecretName
+  # storageClass: myStorageClass
+
+  labels: {}
+  #   foo: bar
+
+## String to partially override node-exporter.fullname template (will maintain the release name)
+##
+# nameOverride:
+
+## String to fully override node-exporter.fullname template
+##
+# fullnameOverride:
+
+## Role Based Access
+## Ref: https://kubernetes.io/docs/admin/authorization/rbac/
+##
+rbac:
+  create: true
+
+  ## RBAC API version
+  ##
+  apiVersion: v1beta1
+
+  ## Podsecuritypolicy
+  ##
+  pspEnabled: true
+
+## Service account for Node Exporter to use.
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+##
+serviceAccount:
+  ## Specifies whether a ServiceAccount should be created
+  ##
+  create: true
+  ## The name of the ServiceAccount to use.
+  ## If not set and create is true, a name is generated using the node-exporter.fullname template
+  # name:
+
+## Bitnami Node Exporter image version
+## ref: https://hub.docker.com/r/bitnami/node-exporter/tags/
+##
+image:
+  registry: docker.io
+  repository: bitnami/node-exporter
```

Comment:
Is it adjustable in any way for using Epiphany's offline mode?
Should we even care about offline mode at this point, WDYT? :thinking:

Reply:
We don't care of offline mode right now. It will be feature in the future :)